src/components/app.js

```diff
@@ -108,8 +108,8 @@ class App extends Component {
   }
 
   componentDidUpdate() {
-    if (this.props.networkId !== undefined && !featuredhiddenListingsFetched) {
-      featuredhiddenListingsFetched = true
+    if (this.props.networkId !== null && !this.featuredhiddenListingsFetched) {
```

Comment:
Just a js question: Is it considered better these days to explicitly test for `null` vs `undefined` vs. something like
```
if (this.props.networkId && !this.featuredhiddenListingsFetched) {
```
I know people are better about using strict comparison operators now.

Reply:
In this case I compared it to null because default value of `networkId` is null in the reducer: https://github.com/OriginProtocol/origin-dapp/blob/master/src/reducers/App.js#L17

Other than that I guess your code would also work fine. I just don't like the implicit conversions javascript does where `if (variable)` that variable could be: 
- null
- undefined
- 0
- false
- empty string
- NaN

and it would all convert to false. I guess this way is more informative to the one reading the code what is the default value instead of being one of 6 above. 

On the other hand the code in your comment is less cluttered. There were probably a bunch of flame wars about this topic 🚒

Reply:
Thanks! I'm returning to the js world after years away and catching up on what has been settled, and what flame wars are still burning! 🔥